l10n_it_partially_deductible_vat/__openerp__.py

```diff
@@ -24,27 +24,25 @@
 ##############################################################################
 
 {
-    "name" : "Italy - Partially Deductible VAT",
-    "version" : "0.1",
-    "depends" : ['account'],
-    "author" : "OpenERP Italian Community,Odoo Community Association (OCA)",
+    "name": "Italy - Partially Deductible VAT",
+    "version": "0.1",
+    "depends": ['account'],
+    "author": "OpenERP Italian Community,Odoo Community Association (OCA),"
+              "SimplERP srl",
     "description": """
```

Comment:
@eLBati i noticed that odoo complains missing description keys. so i guess it would be better to keep it even if empty. never checked what the others do actually.